dotnet/src/Autogen.Ollama/ModelReplyOptions.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// ModelReplyOptions.cs
+
+using System.Text.Json.Serialization;
+
+namespace Autogen.Ollama;
+
+//https://github.com/ollama/ollama/blob/main/docs/modelfile.md#valid-parameters-and-values
+public class ModelReplyOptions
```

Comment:
Can you move this class to DTOs as well?

Reply:
done :)